instrumentor/patch/annotation.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Credits: https://github.com/keyval-dev/odigos
+*/
+
+package patch
+
+import (
+	"context"
+	"github.com/logzio/kubernetes-instrumentor/api/v1alpha1"
+	"github.com/logzio/kubernetes-instrumentor/common/consts"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	goclient "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"log"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ApplicationTypeAnnotation  = "logz.io/application_type"
```

Comment:
Can we add the annotations to the common folder, have them public, and only refer to them, instead of defining them again and again in different folders?